internal/app/singularity/pull_darwin.go

```diff
@@ -39,7 +39,7 @@ func pullRun(cmd *cobra.Command, args []string) {
 		}
 	case ShubProtocol:
 		// would use libexec.PullShubImage but it pulls in build.X
-		err := shub.DownloadImage(name, args[i], force)
+		err := shub.DownloadImage(name, args[i], force, noHTTPS)
```

Comment:
This was missed in a previous commit - likely because we do not include macOS builds in the CircleCI matrix.